class ContactNotFound(Exception):
    pass


class ContactUnexpectedResultError(Exception):
    pass
